distribution_strategy/parameter_server_training/resnet_cifar_ps_strategy.py

```diff
@@ -0,0 +1,297 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+# Lint as: python3
+"""ResNet Cifar + ParameterServerStrategy example.
+"""
+import os
+from datetime import datetime
+import multiprocessing
+from absl import app
+from absl import flags
+from absl import logging
+import portpicker
+import tensorflow as tf
+from tensorflow_models.official.benchmark.models import cifar_preprocessing
+from tensorflow_models.official.benchmark.models import resnet_cifar_model
+from tensorflow_models.official.vision.image_classification.resnet import common as img_class_common
+
+flags.DEFINE_string("checkpoint_dir", "gs://cifar10_ckpt/",
+                    "Directory for writing model checkpoints.")
+flags.DEFINE_string("data_dir", "gs://cifar10_data/",
+                    "Directory for Resnet Cifar model input. Follow the "
+                    "instruction here to get Cifar10 data: "
+                    "https://github.com/tensorflow/models/tree/r1.13.0/official/resnet#cifar-10")
```

Comment:
redundant new line?

Reply:
Split the help argument into multiple lines for readability; they are displayed as concatenated if help cmdline arg is passed.